opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -406,6 +406,18 @@ namespace Opm {
         //compute well guideRates
         const auto& comm = ebosSimulator_.vanguard().grid().comm();
         WellGroupHelpers::updateGuideRatesForWells(schedule(), phase_usage_, reportStepIdx, simulationTime, well_state_, comm, guideRate_.get());
+        updateAndCommunicateGroupData();
+        // Compute initial well solution for new wells
+        for (auto& well : well_container_) {
+            const uint64_t effective_events_mask = ScheduleEvents::WELL_STATUS_CHANGE
+                                                 + ScheduleEvents::NEW_WELL;
+            const bool event = report_step_starts_ && schedule().hasWellGroupEvent(well->name(), effective_events_mask, reportStepIdx);
```

Comment:
I got a compilation error related to `hasWellGroupEvent` in one computer. I could not find the definition of the function `hasWellGroupEvent`.  Not sure how it got compiled on Jenkins. Let me try again.

Reply:
It is confirmed in another setup. Please fix the compilation here.

Reply:
> I got a compilation error related to `hasWellGroupEvent` in one computer.

The function was removed in @joakim-hove's recent PR OPM/opm-common#2194 ([diff](https://github.com/OPM/opm-common/pull/2194/files#diff-c4252a41c30bbb4d8fefb4e1c397779a3ec152c113be77ea238286d562bb8cb8L266)).  That PR in turn is part of a larger `Schedule` object refactoring (OPM/opm-common#2176).  This PR must be rebased onto the current master sources updated in the style of PR #3010.

Reply:
I guess it needs an rebase. See the recently merged
https://github.com/OPM/opm-simulators/pull/3010